Detections/DuoSecurity/TrustMonitorEvent.yaml

```diff
@@ -0,0 +1,30 @@
+id: 8dcf7238-a7d0-4cfd-8d0c-b230e3cd9182
+name: Trust Monitor Event
+description: |
+  'This query identifies when a new trust monitor event is detected.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: DuoSecurity
+    dataTypes: 
+      - DuoSecurityTrustMonitor_CL
+queryFrequency: 5m
+queryPeriod: 5m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+query: |
+
+  let timeframe = ago(5m);
+  DuoSecurityTrustMonitor_CL
+  | where TimeGenerated >= timeframe
```

Comment:
Should there be a filter here for a certain type of event or is everything in this table something that should be alerted on?

Reply:
These are detections in Duo Security (https://duo.com/docs/trust-monitor) they do monitoring of their logs and generate alerts in Trust Monitor so they should be investigated in Sentinel